### ISSUE TITLE
Hello, i train model3v2 on SemEval 2010 task8 dataset, but cannot get the 84% f1?

### ISSUE THREAD

**Issue by tangm00** [Sshanu/Relation-Extraction#4]
what can i do to reach the performance? thank you

**Comment by Sshanu** [comment]
You may reach that accuracy by applying these steps:
1. First, feed the data in the model using bootstrap
2. Use dropout in LSTM
3. I have not optimized the hyperparameters well 
4. Train the POS and Dependency embeddings by Skip Gram Model or any other technique.
Then use these embeddings for initializing the embeddings in the model.
Try these it will help in increasing the accuracy.

**Comment by tangm00** [comment]
Thank you very much! You mean using bootstrap resampling methods for the training data in each batch?

**Comment by Sshanu** [comment]
Yeah

**Comment by tangm00** [comment]
Thank you.

**Comment by damao666** [comment]
> what can i do to reach the performance? thank you

Can you provide me with a working code? Every time I run, I report an error.

Thank you very much.
